doc/quickstart.rst

```diff
@@ -40,20 +40,55 @@ Directories and Permissions
 In order for ``suricata-update`` to function, the following
 permissions are required:
 
-* Directory /etc/suricata: read access
+* Directory /etc/suricata: read/write access
 * Directory /var/lib/suricata/rules: read/write access
 * Directory /var/lib/suricata/update: read/write access
 
 One option is to simply run ``suricata-update`` as root or with
 ``sudo``.
 
 .. note:: It is recommended to create a ``suricata`` group and setup
-          the above directories with the correction permissions for
+          the above directories with the correct permissions for
           the ``suricata`` group then add users to the ``suricata``
           group.
 
-	  More documentation will be provided about this, including a
-	  tool to verify and maybe setup the permissions.
+Steps to setup the above directories with the correct permissions:
+
+First, create a group ``suricata``::
+
+    sudo groupadd suricata
+
+Next, change the group of the directories and its files recursively::
+
+    sudo chgrp -R suricata /etc/suricata
+    sudo chgrp -R suricata /var/lib/suricata/rules
+    sudo chgrp -R suricata /var/lib/suricata/update
+
+.. note:: Do not use the path ``/etc/suricata`` and ``/var/lib`` for
+	        example, since it could be any other path that user defines
+	        during configuration and is by default ``/usr/local``. Please
+	        check your configuration directory for the path and use that
+	        accordingly.
+
+Setup the directories with the correct permissions for the ``suricata``
+group::
+
+    sudo chmod -R g+rw /etc/suricata/
```

Comment:
The group only needs read-only access to /etc/suricata.  So:
```
sudo chmod -R g+r /etc/suricata
```
should be enough.